README.md

```diff
@@ -13,13 +13,13 @@ The sample files for deployment on kubernetes are available in the `deployment-e
 
```

Comment:
Minor changes and loads of linting happened in this file, feel free to ignore. 😄